modules/340-monitoring-kubernetes-control-plane/monitoring/prometheus-rules/kube-etcd3.yaml

```diff
@@ -115,22 +115,6 @@
         2. High CPU usage on the Node.
       summary: Synching (fsync) WAL files to disk is slow.
 
-  - alert: KubeEtcdDatabaseSizeCloseToTheLimit
```

Comment:
We agreed that this is not important because if Deckhouse does not manage etcd (e.g., managed cloud services), it can do nothing about the size of the database.